flexmeasures/api/sunset/__init__.py

```diff
@@ -0,0 +1,69 @@
+from flask import Flask, Blueprint
+
```

Comment:
I want us to add (crisp, one-oline) module docstrings from now on in `__init__.py` files. This helps reading, and also makes the new code documentation better to browse. E.g.:

```
"""
A place to keep all routes to endpoints which previously existed and are now sunset.
"""
```